### PR TITLE
Conform to upstream dir layout structure

### PR DESCRIPTION
**Pull request by LiliC**
Currently our directory structure is not aligned with upstream Kubernetes API directory layout. The upstream code generator tools make assumptions on the specific directory layout.

I also renamed the `cr` to `habitat`, as this makes more sense now that the `habitat` directory is gone. And it is also what all the upstream and other operators do.

Closes https://github.com/kinvolk/habitat-operator/issues/145.

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -16,7 +16,8 @@
 package client
 
 import (
-	crv1 "github.com/kinvolk/habitat-operator/pkg/habitat/apis/cr/v1"
+	habv1 "github.com/kinvolk/habitat-operator/pkg/apis/habitat/v1"
```
Comment (LiliC) [inline pkg/client/client.go]: One thing I would like to get feedback on:
I think we should move to `v1beta1`, as `v1` is considered stable and I am not sure we are there yet with our API. The change doesn't need to happen in this PR. Just collecting feedback here.

Comment (schu) [inline pkg/client/client.go]: Nit: in other files you use `habitatv1`, not `habv1`. Maybe change to `habitatv1` here.

Comment (LiliC) [reply pkg/client/client.go]: Actually we already had naming pattern established with `habitat/client` -> `habclient`, so I just changed this to `habv1` instead.

(Sorry for the force push.)

Comment (krnowak) [reply pkg/client/client.go]: The v1beta looks good, I think. But that would need some planning when we can declare the API as stable.

File: cmd/habitat-operator/main.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 
-	habitatclient "github.com/kinvolk/habitat-operator/pkg/habitat/client"
-	habitatcontroller "github.com/kinvolk/habitat-operator/pkg/habitat/controller"
+	habitatclient "github.com/kinvolk/habitat-operator/pkg/client"
+	habitatcontroller "github.com/kinvolk/habitat-operator/pkg/controller"
```
Comment (krnowak) [inline cmd/habitat-operator/main.go]: habclient and habcontroller?

**Comment by LiliC** [comment]
@krnowak Addressed your comment. PTALA.

**Pull request merged**